packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -132,7 +129,9 @@ class ConfigManager implements IConfigManager {
       state.source.value = {
         id: res.source.id,
       };
+
       // set device mode destination related information in state
+      // TODO: should this not only contain the non cloud destinations?
```

Comment:
Yes.

Reply:
lets discuss this one @MoumitaM and @saikumarrs